superset-frontend/tsconfig.json

```diff
@@ -20,7 +20,13 @@
     "sourceMap": true,
     "strictNullChecks": true,
     "suppressImplicitAnyIndexErrors": true,
-    "target": "es5"
+    "target": "esnext"
```

Comment:
Since Babel will transform the script again anyway, set build target to the fanciest possible so TypeScript compiler can do less work.